Tests/SonarScanner.MSBuild.Shim.Test/ScannerEngineInputGeneratorTest.cs

```diff
@@ -41,11 +42,14 @@ public void Constructor_Null_Throws()
         var cnfg = new AnalysisConfig();
         var rntm = runtime;
         var rvsf = new RoslynV1SarifFixer(runtime.Logger);
-        FluentActions.Invoking(() => new ScannerEngineInputGenerator(null, rntm)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("analysisConfig");
-        FluentActions.Invoking(() => new ScannerEngineInputGenerator(cnfg, null)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("runtime");
-        FluentActions.Invoking(() => new ScannerEngineInputGenerator(cnfg, null, null, null)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("runtime");
-        FluentActions.Invoking(() => new ScannerEngineInputGenerator(cnfg, rntm, null, null)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("fixer");
-        FluentActions.Invoking(() => new ScannerEngineInputGenerator(cnfg, rntm, rvsf, null)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("additionalFilesService");
+        var cmds = new ListPropertiesProvider();
+        FluentActions.Invoking(() => new ScannerEngineInputGenerator(null, cmds, rntm)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("analysisConfig");
+        FluentActions.Invoking(() => new ScannerEngineInputGenerator(cnfg, cmds, null)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("runtime");
+        FluentActions.Invoking(() => new ScannerEngineInputGenerator(cnfg, null, rntm)).Should().ThrowExactly<ArgumentNullException>().WithParameterName("cmdLineArgs");
```

Comment:
These 3 are in a wrong order. The `null` in the table should form a diagonal, to make it regular and easier to read